src/build.d

```diff
@@ -883,6 +883,19 @@ void parseEnvironment()
     }
 }
 
+/**
+Get the value of a build variable that should always be 0, 1 or empty.
+*/
+bool getNumberedBool(const string[string] env, string varname)
```

Comment:
Why not `getBool`?

Reply:
`getBool` sounds like it's expecting `empty`, `"true"` or `"false"` rather than `empty`, `"0"` or `"1"`.